data/json/hobbies.json

```diff
@@ -1,4 +1,59 @@
 [
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "driving_license",
+    "name": "Driving License",
+    "description": "You got your driving license and know how to drive a car, change the oil and tires, and other basics.",
```

Comment:
> change the oil and tires

Does this count as a basic life skill when most people don't know how to do it?
(~~Damn it humanity~~)

Reply:
https://www.prnewswire.com/news-releases/national-survey-reveals-large-majority-of-americans-know-how-to-change-a-tire-300772097.html#:~:text=78%25%20of%20Gen%20Xers%20and,of%20the%20Silent%20%26%20Greatest%20Generations.

Most people can change a tire and then just complain memishly about how they're the only ones who can do it.